js/utils/Translator.js

```diff
@@ -0,0 +1,393 @@
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define('Translator', factory);
+  }
+  else if (typeof module === 'object' && module.exports) {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory();
+  }
+  else {
+    root.Translator = factory();
+  }
+}(this, function () {
```

Comment:
I don't think we need the block above since we use Webpack for bundling. You can simply add this at the bottom:

```js
export default Pluralizer;
```

and then import it on the other script:

```js
import Pluralizer from 'utils/pluralizer.js'
```